### DISCUSSION TITLE
No extruding

### DISCUSSION THREAD

**Discussion by TobyCWood** [FormerLurker/Octolapse#834] [General]
I'm trying to get Octolapse to work first with my Anycubic Predator (which has _weird_ FW) before moving over to my Duet/RRF machines. Everything works fine including Timelapse in Octoprint. But when I move to OctoLapse the extruder stepper does not function. I'm guessing it's in the settings but I really haven't a clue. Probably the printer profile and FW settings. 
Any ideas?

**Comment by FormerLurker** [comment]
Two possibilities come to mind:  
1.  Either test mode is enabled
2. 'G90/G91 Influences Extruder' setting is incorrect in your printer profile.

**Comment by TobyCWood** [comment]
It was test mode. Doh!! Thanks for the help. Awesome product. I’ll be talking about it on the podcast…
3d Printing Today
